src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,45 @@
+.nav {
+  display: flex;
+  height: 60px;
+
+  &__list {
+    display: flex;
+    height: 60px;
+    padding: 0 40px;
+  }
+
+  &__item {
+    list-style: none;
+    margin: auto;
+  }
+
+  &__link {
+    display: flex;
+    margin: auto 10px;
+    text-transform: uppercase;
+    align-items: center;
+    text-decoration: none;
+    color: #{$color-title};
+    height: 60px;
+  }
+
+  .is-active {
```

Comment:
let's create active modifier for nav links and move these styles to those modifier
and use is-active class only in html for tests passing